buffer.go

```diff
@@ -71,6 +71,7 @@ func (b *Buffer) setText(v string) {
 	b.workingLines[b.workingIndex] = v
 
 	if o != v {
+		dummyExecutor("")
```

Comment:
May I have the reason why you call dummyExecutor here? L90 is also.

Reply:
@c-bata The goal is to avoid `SA9003: empty branch (staticcheck)` (https://staticcheck.io/docs/checks#SA9003)

I was looking for a noop and your `prompt.dummyExecutor("")` happened to fit the bill.

Reply:
Thank you for your reply. I'll remove these empty `if` statements in another PR.